src/status_im2/contexts/profile/data/events.cljs

```diff
@@ -0,0 +1,157 @@
+(ns status-im2.contexts.profile.data.events
+  (:require [clojure.string :as string]
+            [native-module.core :as native-module]
+            [quo2.foundations.colors :as colors]
+            [re-frame.core :as re-frame]
+            [react-native.platform :as platform]
+            [status-im.multiaccounts.update.core :as multiaccounts.update]
+            [status-im2.common.theme.core :as theme]
+            [status-im2.constants :as constants]
+            [status-im2.contexts.shell.jump-to.utils :as shell.utils]
+            [status-im2.setup.hot-reload :as hot-reload]
+            [taoensso.timbre :as log]
+            [utils.re-frame :as rf]))
+
+;; TODO(alwx): this function must be removed once the old bottom sheet is completely removed
+(rf/defn hide-legacy-bottom-sheet
+  [{:keys [db]}]
+  {:db                               (assoc db :bottom-sheet/show? false)
+   :dismiss-bottom-sheet-overlay-old nil})
+
+(defn- clean-path
+  [path]
+  (if path
+    (string/replace-first path #"file://" "")
+    (log/warn "[native-module] Empty path was provided")))
+
+(re-frame/reg-fx
+ :profile.data/blank-preview-flag-changed
+ (fn [flag]
+   (native-module/set-blank-preview-flag flag)))
+
+(re-frame/reg-fx
```

Comment:
I think `effects` shouldn't be mixed in the `events` namespace. There's no decision about where to put them, but I think a good start is to create an `contexts/profile/data/effects.cljs`. People already get confused with what's an effect or event and they serve very different purposes. What do you guys think? @flexsurfer @J-Son89 @ulisesmac

Reply:
Sounds good to me

Reply:
yeah i agree, we need a decision, i would vote for having effects.cljs and `:effect/` ns

Reply:
Completely agree!

Reply:
Done